data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as an act of righteous defiance."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
```

Comment:
I love that reference to Nietzsche
Otherwise I'd have said to change it to "I don't believe in Gods"
But since its a ref, keep it!

Reply:
```suggestion
				`	"The Pug fancy themselves gods, but since God is dead..."`
```
Would it be better that way? Caus idk about the so....